metadata-ingestion/source_docs/snowflake.md

```diff
@@ -188,6 +192,11 @@ This source only does usage statistics. To get the tables, views, and schemas in
 
 :::
 
+### Caveats
+- Some of the features are only available in the Snowflake Enterprise Edition. This docs has notes mentioning where this applies.
+- The underlying Snowflake views that we use to get metadata have a [latency of 45 minutes to 3 hours](https://docs.snowflake.com/en/sql-reference/account-usage.html#differences-between-account-usage-and-information-schema). So we would not be able to get very recent metadata in some cases like queries you ran within that time period etc..
+- If there is any [incident going on for Snowflake](https://status.snowflake.com/) we will not be able to get the metadata until that incident is resolved.
```

Comment:
Do we need this? This should be true for any other cloud data warehouse.

Reply:
I would like to add their status pages too whenever I get the chance. Better for people to remember the status pages, check them and then say there is a bug in the connector. I don't think by default people look for status pages. Non-technical users won't know that a status page even exists.